README.md

```diff
@@ -17,16 +17,24 @@
 
-## IPLD Repos
-
-Check the current IPLD Work at the following repos:
-- Content ID / CID: https://github.com/ipfs/go-cid
-- Node interface for IPLD: https://github.com/ipfs/go-ipld-format
-- CBOR IPLD Node: https://github.com/ipfs/go-ipld-cbor
-- Git IPLD Node: https://github.com/ipfs/go-ipld-git
-- Ethereum IPLD Node: https://github.com/ipfs/go-ipld-eth
-- Bitcoin IPLD Node: https://github.com/ipfs/go-ipld-btc
-- Zcash IPLD Node: https://github.com/ipfs/go-ipld-zcash
+## Packages
+
+> This table is generated using the module [`package-table`](https://github.com/ipfs-shipyard/package-table) with `package-table --data=package-list.json`.
+
+Listing of dependencies from the IPLD ecosystem.
+
+| Name | CI/Travis | CI/Jenkins | Coverage |
+| ---------|---------|---------|--------- |
+| **IPLD Formats** |
+| [`ipld-format`](//github.com/ipfs/go-ipld-format) | [![Travis CI](https://travis-ci.org/ipfs/go-ipld-format.svg?branch=master)](https://travis-ci.org/ipfs/go-ipld-format) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=ipfs/go-ipld-format/master)](https://ci.ipfs.team/job/ipfs/job/go-ipld-format/job/master/) | [![codecov](https://codecov.io/gh/ipfs/go-ipld-format/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ipld-format) |
+| [`ipld-cbor`](//github.com/ipfs/go-ipld-cbor) | [![Travis CI](https://travis-ci.org/ipfs/go-ipld-cbor.svg?branch=master)](https://travis-ci.org/ipfs/go-ipld-cbor) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=ipfs/go-ipld-cbor/master)](https://ci.ipfs.team/job/ipfs/job/go-ipld-cbor/job/master/) | [![codecov](https://codecov.io/gh/ipfs/go-ipld-cbor/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ipld-cbor) |
+| [`ipld-git`](//github.com/ipfs/go-ipld-git) | [![Travis CI](https://travis-ci.org/ipfs/go-ipld-git.svg?branch=master)](https://travis-ci.org/ipfs/go-ipld-git) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=ipfs/go-ipld-git/master)](https://ci.ipfs.team/job/ipfs/job/go-ipld-git/job/master/) | [![codecov](https://codecov.io/gh/ipfs/go-ipld-git/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ipld-git) |
+| [`ipld-eth`](//github.com/ipfs/go-ipld-eth) | [![Travis CI](https://travis-ci.org/ipfs/go-ipld-eth.svg?branch=master)](https://travis-ci.org/ipfs/go-ipld-eth) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=ipfs/go-ipld-eth/master)](https://ci.ipfs.team/job/ipfs/job/go-ipld-eth/job/master/) | [![codecov](https://codecov.io/gh/ipfs/go-ipld-eth/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ipld-eth) |
+| [`ipld-btc`](//github.com/ipfs/go-ipld-btc) | [![Travis CI](https://travis-ci.org/ipfs/go-ipld-btc.svg?branch=master)](https://travis-ci.org/ipfs/go-ipld-btc) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=ipfs/go-ipld-btc/master)](https://ci.ipfs.team/job/ipfs/job/go-ipld-btc/job/master/) | [![codecov](https://codecov.io/gh/ipfs/go-ipld-btc/branch/master/graph/badge.svg)](https://codecov.io/gh/ipfs/go-ipld-btc) |
```

Comment:
For the ones that there is no CI, can you replace the wonky badge by a `N/A`?

Reply:
Maybe we could do that on package-table. There must be APIs for that :)

Reply:
yep. In fact, you just have to do a HEAD request for the image and if it returns a 404, then there is no CI running there :)

Reply:
Done1